### PR TITLE
[DO NOT MERGE] removing articles from URL

### PR DESCRIPTION
**Pull request by mairaw**
Need to test if the configuration I've applied on docfx.json was done correctly.

### CODE REVIEW

File: samples/core/console-apps/Fibonacci/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Fibonacci Sample
 ================
 
-This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/articles/core/tutorials/using-with-xplat-cli)
+This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/core/tutorials/using-with-xplat-cli)
```
Comment (BillWagner) [inline samples/core/console-apps/Fibonacci/README.md]: Wouldn't these be more resilient if we targeted the markdown using a relative link?
e g: ../../tutorials/using-with-xplat-cli.md

File: samples/core/console-apps/FibonacciBetter/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Better Fibonacci Sample
 ================
 
-This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/articles/core/tutorials/using-with-xplat-cli)
+This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/core/tutorials/using-with-xplat-cli)
```
Comment (BillWagner) [inline samples/core/console-apps/FibonacciBetter/README.md]: relative link?

File: samples/core/console-apps/FibonacciBetterMsBuild/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Better Fibonacci Sample
 ================
 
-This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/articles/core/tutorials/using-with-xplat-cli)
+This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/core/tutorials/using-with-xplat-cli)
```
Comment (BillWagner) [inline samples/core/console-apps/FibonacciBetterMsBuild/README.md]: relative link?

File: samples/core/console-apps/Hello/README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Hello Sample
 ================
 
-This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/articles/core/tutorials/using-with-xplat-cli)
+This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/core/tutorials/using-with-xplat-cli)
```
Comment (BillWagner) [inline samples/core/console-apps/Hello/README.md]: relative link?

File: samples/core/console-apps/HelloMsBuild/README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 Hello Sample
 ================
 
-This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/articles/core/tutorials/using-with-xplat-cli)
+This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/core/tutorials/using-with-xplat-cli)
```
Comment (BillWagner) [inline samples/core/console-apps/HelloMsBuild/README.md]: relative link?

File: samples/core/console-apps/NewTypes/README.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 NewTypes Pets Sample
 ================
 
-This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/articles/core/tutorials/using-with-xplat-cli)
+This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/core/tutorials/using-with-xplat-cli)
```
Comment (BillWagner) [inline samples/core/console-apps/NewTypes/README.md]: relative link?

File: samples/core/console-apps/NewTypesMsBuild/README.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 NewTypes Pets Sample
 ================
 
-This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/articles/core/tutorials/using-with-xplat-cli) for creating .NET Core Console Applications. Please see that topic for detailed steps on the code for this sample.
+This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/core/tutorials/using-with-xplat-cli) for creating .NET Core Console Applications. Please see that topic for detailed steps on the code for this sample.
```
Comment (BillWagner) [inline samples/core/console-apps/NewTypesMsBuild/README.md]: relative link?

File: samples/core/console-apps/fibonacci-msbuild/README.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 Hello Sample
 ================
 
-This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/articles/core/tutorials/using-with-xplat-cli)
+This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/core/tutorials/using-with-xplat-cli)
```
Comment (BillWagner) [inline samples/core/console-apps/fibonacci-msbuild/README.md]: relative link?

File: samples/core/getting-started/unit-testing-using-dotnet-test/README.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # Unit testing using dotnet test sample
 
-This sample is part of the [unit testing tutorial](https://docs.microsoft.com/dotnet/articles/core/testing/unit-testing-with-dotnet-test) for creating applications with unit tests included. See that topic for detailed steps on the code for this sample.
+This sample is part of the [unit testing tutorial](https://docs.microsoft.com/dotnet/core/testing/unit-testing-with-dotnet-test) for creating applications with unit tests included. See that topic for detailed steps on the code for this sample.
```
Comment (BillWagner) [inline samples/core/getting-started/unit-testing-using-dotnet-test/README.md]: relative link?

File: samples/core/getting-started/unit-testing-using-mstest/README.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 # Unit testing using MSTest sample
 
-This sample is part of the [unit testing tutorial](https://docs.microsoft.com/dotnet/articles/core/testing/unit-testing-with-mstest) for creating applications with unit tests included. See that topic for detailed steps on the code for this sample.
+This sample is part of the [unit testing tutorial](https://docs.microsoft.com/dotnet/core/testing/unit-testing-with-mstest) for creating applications with unit tests included. See that topic for detailed steps on the code for this sample.
```
Comment (BillWagner) [inline samples/core/getting-started/unit-testing-using-mstest/README.md]: relative link?

File: samples/core/hosting/README.md [hunk 11]
```diff
@@ -1,7 +1,7 @@
 .NET Core Hosting Sample
 ==================================
 
-This sample demonstrates a very simple .NET Core host. It is part of the [.NET Core hosting tutorial](https://docs.microsoft.com/en-us/dotnet/articles/core/tutorials/netcore-hosting). Please see that topic for a more detailed explanation of this sample.
+This sample demonstrates a very simple .NET Core host. It is part of the [.NET Core hosting tutorial](https://docs.microsoft.com/en-us/dotnet/core/tutorials/netcore-hosting). Please see that topic for a more detailed explanation of this sample.
```
Comment (BillWagner) [inline samples/core/hosting/README.md]: relative link?

File: samples/core/hosting/README.md [hunk 12]
```diff
@@ -21,7 +21,7 @@ To build this sample, just use msbuild: `msbuild.exe SampleHost.vcxproj`. Be sur
 
 To run the host, you will need a .NET Core-targeted app to run with it.
 
-1. Build a simple .NET Core app (like 'hello world') as a [self-contained](https://docs.microsoft.com/en-us/dotnet/articles/core/deploying/#self-contained-deployments-scd) app (since this host looks for coreclr.dll next to the target app) and publish it.
+1. Build a simple .NET Core app (like 'hello world') as a [self-contained](https://docs.microsoft.com/en-us/dotnet/core/deploying/#self-contained-deployments-scd) app (since this host looks for coreclr.dll next to the target app) and publish it.
```
Comment (BillWagner) [inline samples/core/hosting/README.md]: relative link?

File: samples/csharp/delegates-and-events/README.md [hunk 13]
```diff
@@ -1,7 +1,7 @@
 C# Delegates and Events Sample
 ================
 
-This sample is created during the [Delegates and Events topic](https://docs.microsoft.com/dotnet/articles/csharp/delegates-events)
+This sample is created during the [Delegates and Events topic](https://docs.microsoft.com/dotnet/csharp/delegates-events)
```
Comment (BillWagner) [inline samples/csharp/delegates-and-events/README.md]: relative link?

File: samples/csharp/expression-trees/README.md [hunk 14]
```diff
@@ -1,7 +1,7 @@
 C# Expression Trees Sample
 ================
 
-This sample is created during the [Expression Trees topic](https://docs.microsoft.com/dotnet/articles/csharp/expression-trees)
+This sample is created during the [Expression Trees topic](https://docs.microsoft.com/dotnet/csharp/expression-trees)
```
Comment (BillWagner) [inline samples/csharp/expression-trees/README.md]: relative link?

File: samples/csharp/getting-started/console-linq/README.md [hunk 15]
```diff
@@ -1,6 +1,6 @@
 # C# LINQ Sample
 
-This sample is created during the [Working with LINQ tutorial](https://docs.microsoft.com/dotnet/articles/csharp/tutorials/working-with-linq) for learning C# features. Please see that topic for detailed steps on the code for this sample.
+This sample is created during the [Working with LINQ tutorial](https://docs.microsoft.com/dotnet/csharp/tutorials/working-with-linq) for learning C# features. Please see that topic for detailed steps on the code for this sample.
```
Comment (BillWagner) [inline samples/csharp/getting-started/console-linq/README.md]: relative link?

File: samples/csharp/getting-started/console-teleprompter/README.md [hunk 16]
```diff
@@ -1,7 +1,7 @@
 C# Console Application Sample
 ================
 
-This sample is created during the [Console Application Tutorial](https://docs.microsoft.com/dotnet/articles/csharp/tutorials/console-teleprompter)
+This sample is created during the [Console Application Tutorial](https://docs.microsoft.com/dotnet/csharp/tutorials/console-teleprompter)
```
Comment (BillWagner) [inline samples/csharp/getting-started/console-teleprompter/README.md]: relative link?

File: samples/csharp/getting-started/console-webapiclient/README.md [hunk 17]
```diff
@@ -1,7 +1,7 @@
 C# REST Client Sample
 ================
 
-This sample is created during the [REST client tutorial](https://docs.microsoft.com/dotnet/articles/csharp/tutorials/console-webapiclient)
+This sample is created during the [REST client tutorial](https://docs.microsoft.com/dotnet/csharp/tutorials/console-webapiclient)
```
Comment (BillWagner) [inline samples/csharp/getting-started/console-webapiclient/README.md]: relative link?

File: samples/csharp/indexers/README.md [hunk 18]
```diff
@@ -1,7 +1,7 @@
 C# Indexers Sample
 ================
 
-This sample is created during the [Indexers](https://docs.microsoft.com/dotnet/articles/csharp/indexers)
+This sample is created during the [Indexers](https://docs.microsoft.com/dotnet/csharp/indexers)
```
Comment (BillWagner) [inline samples/csharp/indexers/README.md]: relative link?

File: samples/csharp/iterators/README.md [hunk 19]
```diff
@@ -1,7 +1,7 @@
 C# Iterators Sample
 ================
 
-This sample is created during the [Iterators topic](https://docs.microsoft.com/dotnet/articles/csharp/iterators)
+This sample is created during the [Iterators topic](https://docs.microsoft.com/dotnet/csharp/iterators)
```
Comment (BillWagner) [inline samples/csharp/iterators/README.md]: relative link?

File: samples/csharp/versioning/new/README.md [hunk 20]
```diff
@@ -1,7 +1,7 @@
 C# Versioning Sample
 ================
 
-This sample is created during the [Versioning](https://docs.microsoft.com/dotnet/articles/csharp/versioning)
+This sample is created during the [Versioning](https://docs.microsoft.com/dotnet/csharp/versioning)
```
Comment (BillWagner) [inline samples/csharp/versioning/new/README.md]: relative link?

File: samples/csharp/versioning/override/README.md [hunk 21]
```diff
@@ -1,7 +1,7 @@
 C# Versioning Sample
 ================
 
-This sample is created during the [Versioning](https://docs.microsoft.com/dotnet/articles/csharp/versioning)
+This sample is created during the [Versioning](https://docs.microsoft.com/dotnet/csharp/versioning)
```
Comment (BillWagner) [inline samples/csharp/versioning/override/README.md]: relative link?

File: samples/framework/libraries/README.md [hunk 22]
```diff
@@ -1,6 +1,6 @@
 # .NET Core Library Samples
 
-These samples are buildable projects whose source is used for code snippets in [the guide for writing cross-platform libraries](https://docs.microsoft.com/dotnet/articles/core/tutorials/libraries).  They can be built and run using the .NET Core toolchain, and are intended to simply demonstrate how to target and build NuGet packages for different targets.  They aren't examples of how you'd build a real, feature-complete library.
+These samples are buildable projects whose source is used for code snippets in [the guide for writing cross-platform libraries](https://docs.microsoft.com/dotnet/core/tutorials/libraries).  They can be built and run using the .NET Core toolchain, and are intended to simply demonstrate how to target and build NuGet packages for different targets.  They aren't examples of how you'd build a real, feature-complete library.
```
Comment (BillWagner) [inline samples/framework/libraries/README.md]: relative link?

File: samples/framework/libraries/README.md [hunk 23]
```diff
@@ -60,16 +60,16 @@ The project under `/net45-compat-library` targets any of the following:
 * Xamarin
 * Mono
 
-It uses the `netstandard1.2` Target Framework Moniker introduced with the [.NET Standard Library](https://docs.microsoft.com/en-us/dotnet/articles/standard/library).
+It uses the `netstandard1.2` Target Framework Moniker introduced with the [.NET Standard Library](https://docs.microsoft.com/en-us/dotnet/standard/library).
```
Comment (BillWagner) [inline samples/framework/libraries/README.md]: relative link?

File: samples/framework/libraries/README.md [hunk 24]
```diff
@@ -60,16 +60,16 @@ The project under `/net45-compat-library` targets any of the following:
...
 
 ## net40-library
 
 **IMPORTANT:** This project requires Windows and the the .NET Framework installed on your machine.
 
-The project under `/net40-library` targets the .NET Framework 4.0 and above.  It also demonstrates how to use [#if](https://docs.microsoft.com/en-us/dotnet/articles/csharp/language-reference/preprocessor-directives/preprocessor-if) directives to multi-target for a .NET 4.0 target.
+The project under `/net40-library` targets the .NET Framework 4.0 and above.  It also demonstrates how to use [#if](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if) directives to multi-target for a .NET 4.0 target.
```
Comment (BillWagner) [inline samples/framework/libraries/README.md]: relative link?

File: samples/framework/libraries/README.md [hunk 25]
```diff
@@ -60,16 +60,16 @@ The project under `/net45-compat-library` targets any of the following:
...
 
 ## pcl-library
 
 **IMPORTANT:** This project requires Windows and the .NET Framework installed on your machine.
 
-The project under `/pcl-library` shows how to target a supported PCL Profile (for example, 344).  It shows how to structure the `Library.csproj` file to allow for targeting a PCL.  It also demonstrates how to use [#if](https://docs.microsoft.com/en-us/dotnet/articles/csharp/language-reference/preprocessor-directives/preprocessor-if) directives and how to define a preprocessor constant, `PORTABLE259` in the `Library.csproj` file.
+The project under `/pcl-library` shows how to target a supported PCL Profile (for example, 344).  It shows how to structure the `Library.csproj` file to allow for targeting a PCL.  It also demonstrates how to use [#if](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if) directives and how to define a preprocessor constant, `PORTABLE259` in the `Library.csproj` file.
```
Comment (BillWagner) [inline samples/framework/libraries/README.md]: relative link?

**Review by BillWagner — CHANGES_REQUESTED** [review]
Overall this looks good.
I'm wondering if there is a way in the global redirection file to catch any links from outside the site that contain 'articles' and route them correctly? If not, where do they go?

If we have time, it would be better to use the relative links inside our own docset. Would you agree?

**Review by rpetrusha — APPROVED** [review]
LGTM also.

File: samples/core/console-apps/Fibonacci/README.md (hunk 1 above)
Comment (mairaw) [inline samples/core/console-apps/Fibonacci/README.md]: @BillWagner these are supposed to be the readme files for the samples. they are not published so if you're looking at this file on your machine, you need the absolute URL.

Comment (BillWagner) [inline samples/core/console-apps/Fibonacci/README.md]: @mairaw I need to get my eyes checked. I thought they were in the main doc set.  Not sure what I was seeing...

**Review by BillWagner — APPROVED** [review]

**Comment by mairaw** [comment]
If I understand the redirection process correctly, any links to https://docs.microsoft.com/en-us/dotnet/articles/* would go to https://docs.microsoft.com/en-us/dotnet/*. This is the part that will be done via Akamai.

We do use relative urls in our repo. The ones I'm fixing were done like this because they are for the readme files. But they would continue to work. I'm just avoiding the redirection by fixing them).

**Comment by mairaw** [comment]
URLs don't have articles any longer on staging but there are some oddities I'm seeing on the TOC. I'll need to investigate why is that happening. 

image

**Comment by mairaw** [comment]
The changes need to be in a branch for the build to fully work (engineering team opened a bug). So merging this one now and opening a new PR to merge against master to test this change.

**Pull request merged**